src/EnvironmentCheckSuite.php

```diff
@@ -184,6 +199,15 @@ public static function register($names, $check, $title = null)
         if (!is_array($names)) {
             $names = array($names);
         }
+
+        /**
+         * Support for omitted namespaces - this will fall back to adding the SilverStripe\EnvironmentCheck\Checks
+         * namespace if one isn't provided. If you are using custom checks, make sure you include a namespace (or "\")
+         */
+        if (is_string($check) && strpos($check, '\\') === false) {
+            $check = __NAMESPACE__ . '\\Checks\\' . $check;
+        }
```

Comment:
~Actually, this doesn't allow anyone else to add their own checks. I'll update this to be a bit more careful.~

Because of the Object class name parsing with formatting options e.g. `MyCheck(123)` it'd be a little annoying to process.

I've left this change here, but made the readme and comment above it clear about what is expected here.